crates/ln-dlc-node/src/tests/just_in_time_channel/channel_close.rs

```diff
@@ -0,0 +1,169 @@
+async fn collab_close() {
+    init_tracing();
+
+    // Arrange
+
+    let payer = Node::start_test_app("payer").await.unwrap();
+    let coordinator = Node::start_test_coordinator("coordinator").await.unwrap();
+    let payee = Node::start_test_app("payee").await.unwrap();
+
+    payer.connect(coordinator.info).await.unwrap();
+    payee.connect(coordinator.info).await.unwrap();
+
+    coordinator.fund(Amount::from_sat(1_000_000)).await.unwrap();
+
+    let payer_outbound_liquidity_sat = 25_000;
+    let coordinator_outbound_liquidity_sat =
+        min_outbound_liquidity_channel_creator(&payer, payer_outbound_liquidity_sat);
+
+    coordinator
+        .open_channel(
+            &payer,
+            coordinator_outbound_liquidity_sat,
+            payer_outbound_liquidity_sat,
+        )
+        .await
+        .unwrap();
+
+    let invoice_amount = 1_000;
+
+    send_interceptable_payment(
+        TestPath::OnlineFunding,
+        &payer,
+        &payee,
+        &coordinator,
+        invoice_amount,
+        Some(JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT),
+    )
+    .await
+    .unwrap();
+
+    close_channel(TestPath::CollabClose, &payee, &coordinator)
+        .await
+        .unwrap();
+}
+
+#[tokio::test]
+#[ignore]
+async fn force_close() {
+    init_tracing();
+
+    // Arrange
+
+    let payer = Node::start_test_app("payer").await.unwrap();
+    let coordinator = Node::start_test_coordinator("coordinator").await.unwrap();
+    let payee = Node::start_test_app("payee").await.unwrap();
+
+    payer.connect(coordinator.info).await.unwrap();
+    payee.connect(coordinator.info).await.unwrap();
+
+    coordinator.fund(Amount::from_sat(1_000_000)).await.unwrap();
+
+    let payer_outbound_liquidity_sat = 25_000;
+    let coordinator_outbound_liquidity_sat =
+        min_outbound_liquidity_channel_creator(&payer, payer_outbound_liquidity_sat);
+
+    coordinator
+        .open_channel(
+            &payer,
+            coordinator_outbound_liquidity_sat,
+            payer_outbound_liquidity_sat,
+        )
+        .await
+        .unwrap();
+
+    let invoice_amount = 1_000;
+
+    send_interceptable_payment(
+        TestPath::OnlineFunding,
+        &payer,
+        &payee,
+        &coordinator,
+        invoice_amount,
+        Some(JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT),
+    )
+    .await
+    .unwrap();
+
+    close_channel(TestPath::ForceClose, &payee, &coordinator)
+        .await
+        .unwrap();
+}
+
+async fn close_channel(
+    test_path: TestPath,
+    payee: &Node<PaymentMap>,
+    coordinator: &Node<PaymentMap>,
+) -> Result<()> {
+    let channel_id = payee
+        .channel_manager
+        .list_usable_channels()
+        .first()
+        .unwrap()
+        .channel_id;
+
+    assert_eq!(payee.get_on_chain_balance()?.confirmed, 0);
+    assert_eq!(payee.get_ldk_balance().available, 1_000);
+    assert_eq!(payee.get_ldk_balance().pending_close, 0);
+
+    if TestPath::ForceClose == test_path {
```

Comment:
:wrench: `match`ing is more idiomatic and generally safer if we end up adding more variants.